app/controllers/api/v1/users_controller.rb

```diff
@@ -14,9 +14,9 @@ def show
   def index
     if params[:user][:search]
       role = if current_user.shop?
-        2
+        "Shipper"
       elsif current_user.shipper?
-        1
+        "Shop"
       end
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.Lint/EndAlignment
**Severity**: WARN
**File**: app/controllers/api/v1/users_controller.rb L20

`end` at 20, 6 is not aligned with `if` at 16, 13.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*